agent.py

```
@@ -0,0 +1,167 @@
+from pydantic_ai import Agent, RunContext, Tool
+from pydantic_ai.messages import ToolReturn
+from pydantic import BaseModel
+from typing import List, Dict
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)